feature/tabsflow/home/src/main/java/com/epmedu/animeal/home/presentation/ui/map/RouteView.kt

```diff
@@ -115,4 +124,25 @@ private fun fetchRoute(
             onRouteResult = onRouteResult
         )
     }
+}
+
+private fun getMapBoxNavigationInitOptions(
+    mapView: MapView,
+    state: HomeState
+): MapBoxRouteInitOptions {
+    val routeLineResources = RouteLineResources.Builder()
+        .originWaypointIcon(R.drawable.ic_your_location)
+
+    state.feedState.feedPoint?.getDrawableRes()?.let { feedingPointDrawable ->
+        routeLineResources.destinationWaypointIcon(feedingPointDrawable)
```

Comment:
This icon is too big and it has wrong color, try to set it from `state.feedingPointState.currentFeedingPoint` instead

Reply:
Hey @Fileek 

I changed the icon feed from `state.feedingPointState.currentFeedingPoint`, however, there is a problem retrieving the image when the user closes the application and opens it again. 

When closes/reopens the application `state.feedingPointState.currentFeedingPoint` returns a `null `icon, resulting in the following map rendering: 

![Screenshot_20230828_150034](https://github.com/AnimealProject/animeal_android/assets/28179406/c5dc952d-c34a-4555-a649-5978b2c08153)

Reply:
Let's then use `feedPoint` from `feedState`, but update its `feedStatus` in the `DefaultFeedingHandler` depending on operation.

Reply:
And please solve the scaling issue:
<details>
<summary>Screenshots:</summary>

| Without route  | With route |
| ------------- | ------------- |
| ![image](https://github.com/AnimealProject/animeal_android/assets/83027107/13c79386-7ede-445a-bee4-1e540004cf18)  | ![image](https://github.com/AnimealProject/animeal_android/assets/83027107/c9eec6bd-5dbc-49b1-b7dd-6c544816d922)  |
</details>

Reply:
@Fileek Unfortunately the API scales the image automatically when setting the `destinationWaypointIcon`
I also tried XML's representing animal states with a smaller dimension but the library scales them to the same dimension even though they are smaller. 

I also checked the Mapbox API for a method to scale the `destinationWaypointIcon`, but they do not permit it :(

Reply:
The icon is not clickable now. It should be clickable to finish feeding.